ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/tasks/PersistBlockTask.java

```diff
@@ -217,13 +217,12 @@ protected void cleanup() {
       final double timeInS = getTaskTimeInSec();
       LOG.info(
           String.format(
-              "Imported #%,d / %d tx / %d om / %,d (%01.1f%%) gas / (%s) in %01.3fs. Peers: %d",
-              block.getHeader().getNumber(),
+              "Imported %s / %d tx / %d om / %,d (%01.1f%%) gas in %01.3fs. Peers: %d",
+              block.getHeader().toLogString(),
```

Comment:
This change is very out of scope for this PR.

Reply:
Agreed. Fixed